### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by VladimirPal**
Update versions of dependencies packages

### CODE REVIEW

**Comment by VladimirPal** [comment]
https://github.com/thejameskyle/marionette-wires/issues/68

File: src/books/show/route.js [hunk 1]
```diff
@@ -6,16 +6,18 @@ import storage from '../storage';
...
     return storage.findAll().then(collection => {
       this.collection = collection;
     });
   },
 
-  onFetch(id) {
-    this.model = this.collection.get(id);
+  onFetch() {
+    this.model = this.collection.get(this.id);
     this.collection.active = this.model;
```
Comment (thejameskyle) [inline src/books/show/route.js]: just move this to the `fetch()` function above inside the `then(collection => { ... });`

File: gulpfile.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
+var babel = require("babel-register")({
+  presets: ['es2015']
```
Comment (thejameskyle) [inline gulpfile.js]: Can you create a `.babelrc` file instead?

**Comment by VladimirPal** [comment]
@thejameskyle Done

File: src/application/application.js [hunk 3]
```diff
@@ -16,32 +12,5 @@ export default Application.extend({
...
-
-  onEnterRoute() {
-    this.transitioning = false;
-    this.$body.scrollTop(0);
-    nprogress.done();
-  },
-
-  onErrorRoute() {
-    this.transitioning = false;
-    nprogress.done(true);
```
Comment (thejameskyle) [inline src/application/application.js]: This shouldn't be part of this PR, only in https://github.com/thejameskyle/marionette-wires/pull/70

**Comment by VladimirPal** [comment]
@thejameskyle done

**Comment by thejameskyle** [comment]
Thanks
